examples/complete/main.tf

```diff
@@ -0,0 +1,50 @@
+##############################################################################
+# Resource Group
+# (if var.resource_group is null, create a new RG using var.prefix)
+##############################################################################
+
+resource "ibm_resource_group" "resource_group" {
+  count = var.resource_group != null ? 0 : 1
+  name  = "${var.prefix}-rg"
+
+}
+
+data "ibm_resource_group" "existing_resource_group" {
+  count = var.resource_group != null ? 1 : 0
+  name  = var.resource_group
+}
+
+
+##############################################################################
+# Create new VPC
+# (if var.vpc_id is null, create a new VPCs using var.prefix)
+##############################################################################
+
+resource "ibm_is_vpc" "vpc" {
```

Comment:
I continue to use `resource "ibm is vpc" "vpc"` for the default example since the security group id used here is the default security group id of the VPC and the `terraform-ibm-landing-zone-vpc` module doesn't have this in the outputs.